From 3b8daa03f9a5ce2641706979c2da2985bb8468a1 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Thu, 28 Sep 2023 16:27:25 +0200
Subject: [PATCH 0557/1141] arm64: dts: st: remove clock of syscfg node

SYSCPU1 is not the clock of SYSCFG IP (this IP is auto gated).

Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: Id063a5df4f515a2c6b4df9c874876302daefdc9d
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/330280
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index b0c1abae1101..67822b26fbd3 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -2366,7 +2366,6 @@ exti-interrupt-map {
 		syscfg: syscon@44230000 {
 			compatible = "st,stm32mp25-syscfg", "syscon";
 			reg = <0x44230000 0x10000>;
-			clocks = <&scmi_clk CK_SCMI_SYSCPU1>;
 			#clock-cells = <1>;
 		};
 
-- 
2.39.2

